CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* **BREAKING:** Fix referrer bug ([PR #2906](https://github.com/alphagov/finder-frontend/pull/2906))
```

Comment:
This is the wrong detail - it should refer to this PR (`https://github.com/alphagov/govuk_publishing_components/pull/3032`) not the one in `finder-frontend`.

Reply:
My mistake, updated now